install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml

```diff
@@ -11,16 +11,23 @@ SEM: {
   class: xt_client.SEM,
   role: null,
   init: {
-    address: "PYRO:Microscope@192.168.31.150:4242",
+    address: "PYRO:Microscope@0.0.0.0:4242",
```

Comment:
I will remove this change before merging

Reply:
Now that all the tests are actually working with the adapter_sim, I think it actually makes sense to change the address to `0.0.0.0`. The `192.168.31.150` is needed for the simulator VM, but that the test cases are only occasionally run with the simulator VM.  But it might be better to do it in a separate PR. 
@pieleric do you agree? I can create a separate PR

Reply:
This is in "hwtest" directory, normally used for testing on real hardware. So I'd suggest to leave it as-is. You could add a comment to indicate that it should be replaced by "localhost:4242" for the simulator.
BTW, I wonder why "0.0.0.0" works. It's not really a valid address.
BTW2, maybe we could skip the "PYRO:Microscope@". It seems to leak some internal implementation details. We could automatically add it when building the URL.

Reply:
>"hwtest" directory, normally used for testing on real hardware. 
I don't think that IP adress corresponds with any of the real hardware, I think both FUMO-Beta (EA when in Delft Zuid) and Helios have another IP adress

Reply:
Still, 0.0.0.0 is the worst as it is not correct. I'm fine if you replace with "localhost", or if you leave the old IP.
Note that the fact you're adding the fib-scanner means that it'll probably fail to start with a VM too, as it doesn't simulate a FIB.

Reply:
@tepals Maybe for another PR we/you can think about what IP adress we want to actually write down in the test cases of xt_client_test? For now I resolve this issue